src/settings_internal.js

```diff
@@ -265,3 +265,5 @@ var POST_JS_FILES = [];
 var PTHREADS = false;
 
 var BULK_MEMORY = false;
+
+var MINIFY_JS = true;
```

Comment:
This looks like a more consequential change.

Reply:
Sorry, somehow that leaked from another PR I'm working on